### PR TITLE
Send Attestation message async

### PR DESCRIPTION
**Pull request by thomaseizinger**
Latest testing revealed that we might be causing a deadlock here by waiting for this message to be delivered.

### CODE REVIEW

**Review by klochowicz — APPROVED** [review]
good cleanup! 👍

File: daemon/src/oracle.rs [hunk 1]
```diff
@@ -258,7 +258,7 @@ impl Actor {
 
         tracing::info!("Fetched new attestation for {id}");
 
-        let _ = self.attestation_channel.send(attestation).await;
+        let _: Result<(), xtra::Disconnected> = self.attestation_channel.send(attestation).await;
```
Comment (bonomat) [inline daemon/src/oracle.rs]: nice

**Review by bonomat — APPROVED** [review]
Let's hope this help.

**Pull request merged**